react/SearchWrapper.tsx

```diff
@@ -116,21 +117,61 @@ const getSearchIdentifier = (
   return query + map + (orderBy ?? '') + (page ?? '')
 }
 
-function getCanonicalMaybeWithPage(canonicalLink: string, page: number) {
-  if (page <= 0) {
+interface GetSearchCanonicalParams {
+  canonicalLink: string
+  page?: number
+  map?: string
+}
+
+function shouldNotIncludeMap(map?: string) {
```

Comment:
This is more of a nitpick, but how about inverting the logic on this function so that you can call it `shouldIncludeMap`? 

I often find a piece of code faster to understand if it doesn't have negative conditional checks, but instead checks for truth values and then takes into account the falsy cases.